mm2src/lp_ordermatch.rs

```diff
@@ -2032,6 +2118,14 @@ impl Orderbook {
             trie_roots: trie_roots_to_request,
         })
     }
+
+    fn orderbook_item_with_proof(&self, order: OrderbookItem) -> Result<OrderbookItemWithProof, ()> {
+        Ok(OrderbookItemWithProof {
+            order,
+            last_message_payload: vec![],
```

Comment:
Are the default `last_message_payload` and `proof` just temporary?

Reply:
Yeah, I added these fields for future use to have a proper validation of the received data.